newrelic_lambda_cli/layers.py

```diff
@@ -96,10 +98,23 @@ def _add_new_relic(config, region, layer_arn, account_id, allow_upgrade):
     if handler != runtime_handler:
         update_kwargs["Environment"]["Variables"]["NEW_RELIC_LAMBDA_HANDLER"] = handler
 
+    if enable_extension:
+        update_kwargs["Environment"]["Variables"][
+            "NEW_RELIC_LAMBDA_EXTENSION_ENABLED"
+        ] = "true"
+
```

Comment:
Since we're assuming here that they didn't have the extension set up before, we need to also grant permission to the managed secret.

Reply:
@MattWhelan Good point. All this issue addresses is the `NEW_RELIC_LAMBDA_EXTENSION_ENABLED` environment variable. But will create an issue for this. Will need to set `NEW_RELIC_LICENSE_KEY` in the event the user isn't using secrets manager as well.